### PR TITLE
image optimizer is created

### PR DESCRIPTION
**Pull request by tarikutk**

### CODE REVIEW

File: public/js/createSrcSet.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+
+fetch("../sourceSets.json").then(response => { return response.json();})
```
Comment (brentonstrine) [inline public/js/createSrcSet.js:2]: In the deploy preview, this is failing to grab the file. That URL resolves to:

https://deploy-preview-226--refcode.netlify.app/sourceSets.json

which is a 404.

Comment (brentonstrine) [reply public/js/createSrcSet.js:2]: Ignore the above, after running the script, that file does exist (see this deploy preview: https://611a67a88af9b53f1184ac09--refcode.netlify.app/sourceSets.json)

**Comment by brentonstrine** [comment]
When this PR is merged,  `npm run optimizeimg` needs to be added to the deploy settings

**Comment by tarikutk** [comment]
ok. so should I include that command in package.json ?

**Comment by brentonstrine** [comment]
Yeah, can we do that and look at the Netlify deploy preview to see if it works. I've removed it from the Netlify config, so when you push the new change it will trigger a new deploy preview and we can see if it works.

**Pull request closed without merging**